.github/workflows/ci.yaml

```diff
@@ -111,80 +111,89 @@ jobs:
-          #            } catch (e) {
-          #              console.log('Failed to remove label. Another job may have already removed it!');
-          #            }
-          #
-          #      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
-          #        with:
-          #          ref: ${{ github.event.pull_request.head.sha }}
-          #          repository: ${{ github.event.pull_request.head.repo.full_name }}
-          #
-          #      - name: Set up GraalVM
-          #        uses: graalvm/setup-graalvm@b11d36630f94ed5864e0a680a979b5afde449e9d
-          #        with:
-          #          version: 'latest'
-          #          java-version: '17'
-          #          components: 'native-image'
-          #          github-token: ${{ secrets.GITHUB_TOKEN }}
-          #
-          #      - id: 'auth'
-          #        name: Authenticate to Google Cloud
-          #        uses: google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d # v1.0.0
-          #        with:
-          #          workload_identity_provider: ${{ secrets.PROVIDER_NAME }}
-          #          service_account: ${{ secrets.SERVICE_ACCOUNT }}
-          #          access_token_lifetime: 600s
-          #
-          #      - id: 'secrets'
-          #        name: Get Secrets
-          #        uses: google-github-actions/get-secretmanager-secrets@7fced8b6579c75d7c465165b38ec29175d9a469c # v1.0.0
-          #        with:
-          #          secrets: |-
-          #            ALLOYDB_INSTANCE_URI:${{ secrets.GOOGLE_CLOUD_PROJECT }}/ALLOYDB_INSTANCE_URI
-          #            ALLOYDB_CLUSTER_PASS:${{ secrets.GOOGLE_CLOUD_PROJECT }}/ALLOYDB_CLUSTER_PASS
-          #
-          #      - run: .kokoro/build.sh
-          #        env:
-          #          ALLOYDB_DB: 'postgres'
-          #          ALLOYDB_USER: 'postgres'
-          #          ALLOYDB_PASS: '${{ steps.secrets.outputs.ALLOYDB_CLUSTER_PASS }}'
-          #          ALLOYDB_INSTANCE_URI: '${{ steps.secrets.outputs.ALLOYDB_INSTANCE_URI }}'
-          #          JOB_TYPE: graalvm17
-          #        shell: bash
-          #
-          #      - name: FlakyBot
-          #        # only run flakybot on periodic (schedule) and continuous (push) events
-          #        if: ${{ (github.event_name == 'schedule' || github.event_name == 'push') && runner.os == 'Linux' && always() }}
-          #        run: |
-          #          curl https://github.com/googleapis/repo-automation-bots/releases/download/flakybot-1.1.0/flakybot -o flakybot -s -L
-          #          chmod +x ./flakybot
-          #          ./flakybot --repo ${{github.repository}} --commit_hash ${{github.sha}} --build_url https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
-          #
+  graalvm17:
+    # run job on periodic (schedule) and continuous (push) events
+    if: "${{ (github.event_name == 'schedule' || github.event_name == 'push') && always() }}"
```

Comment:
Dumb question: what is push? Pushes to PRs and merges to main?

Reply:
A `push` event is created when we merge a PR to the `main` branch as we configured:

```
on:
  push:
    branches:
    - main
```

[official documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-to-specific-branches-occurs)

Reply:
@ttosta-google  Why do we only want this on `push` and `schedule` events and not on pull requests? I also don't think you want the `always()` here...

Reply:
Updated to run on `schedule` as agreed. Removed `always()`